templates/registration/login.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Login</h1>
+    {% if error %}
```

Comment:
The `error` variable is used to display error messages. Ensure that this variable is passed to the template context in the view handling the login, or consider using Django's built-in form error handling to display errors.